modules/docs/arrow-docs/docs/docs/arrow/typeclasses/unalign/README.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: docs
+title: Unalign
+permalink: /docs/arrow/typeclasses/unalign/
+redirect_from:
+  - /docs/typeclasses/unalign/
+---
+
+## Unalign
+
+{:.beginner}
+beginner
+
+The `Unlign` typeclass extends the `Semialign` typeclass with an inverse function to align: It splits an union shape
```

Comment:
```suggestion
The `Unlign` typeclass extends the `Semialign` typeclass with an inverse function to align: It splits a union shape
```

Reply:
thanks, i looked that up and realized i learned that wrong in school back then.

Reply:
Yeah, union is one of the few exceptions to the rules surrounding a/an. Learned it wrong as well :)